resources/authgear/templates/en/web/authflowv2/view_recovery_code.html

```diff
@@ -0,0 +1,52 @@
+{{ template "authflowv2/__page_frame.html" . }}
+{{ define "page-content" }}
+
+  <div class="space-y-4">
+    {{ template "authflowv2/__header.html" . }}
+    <h1 class="screen-title">
+      {{ template "recovery-code-title" }}
+    </h1>
+    <h2 class="screen-description">
+      {{ template "recovery-code-storage-description" }}
+    </h2>
+  </div>
+
+  <div class="my-6 code-block">
+    <p class="code-block__text">{{ range $.RecoveryCodes }}{{ . }}
+{{ end }}</p>
+
+    <code id="copy-button-source" class="hidden">{{ template "__recovery_code.html" . }}</code>
+    <div class="mt-5 w-full grid gap-4 grid-cols-1 tablet:grid-cols-2">
+      <form id="download-form" class="hidden tablet:block" method="post" novalidate target="_blank">
+        {{ $.CSRFField }}
+        <button
+          form="download-form"
+          class="tertiary-btn w-full"
+          type="submit"
+          name="x_action"
+          value="download"
+          data-turbo="false"
+        >
+          {{ template "download-button-label" }}
+        </button>
```

Comment:
If I remember it correctly, in previous discussion we said we will only display download button in desktop.

@fungc-io Please correct me if I am wrong.

Reply:
yes, you are correct

Reply:
@IniZio I guess we should use `IsSupportedMobilePlatform` here